mobile/hooks/useSignOut.ts

```
@@ -0,0 +1,27 @@
+import { Alert } from "react-native";
+
+export const useSignOut = () => {
+  const { signOut } = useClerk();
+
+  const handleSignOut = () => {
+    Alert.alert("Logout", "Are you sure you want to logout?", [
+      { text: "Cancel", style: "cancel" },
+      {
+        text: "Logout",
+        style: "destructive",
+        onPress: () => signOut(),
```

Biome (1.9.4)

[error] This property value named onPress is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named onPress


(lint/suspicious/noDuplicateObjectKeys)